ts/reviewer/images.ts

```diff
@@ -49,3 +49,17 @@ export function preloadAnswerImages(qHtml: string, aHtml: string): void {
         diff.forEach((src) => injectPreloadLink(src, "image"));
     }
 }
+
+export async function maybePreloadImages(html: string): Promise<void> {
+    const srcs = extractImageSrcs(html);
+    await Promise.race([
+        Promise.all(
+            Array.from(srcs).map((src) => {
```

Comment:
Just a minor thing: Isn't `Array.from` redundant as the return value of `extractImageSrcs` is already an array of string?